### PR TITLE
chore: CPO-286 upgrade to Node18 & npm 9

### PR DESCRIPTION
**Pull request by NicholasValiente**
### Issue Link:

CPO-286

### What does it do?

- Update NodeJS to version 18
- Update NPM to version 9 
- Update a number of packages to latest

#### Checklist

- [ ] BREAKING CHANGE - I have/intend to update all places where this package is used.
- [ ] I will squash: PR title conforms to standard commit message format (`feat: XX-1234 ...`)
- [x] I will merge without squashing: All commits conforms to standard commit message format (`feat: XX-1234 ...`)
- [ ] README update

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,8 +42,8 @@
     ]
   },
   "dependencies": {
-    "@actions/core": "^1.10.0",
-    "@actions/github": "^5.1.1",
```
Comment (trend-adam-kinnell) [inline package.json:46]: ❓ Just confirming there were no breaking changes in this update?

**Review by trend-adam-kinnell — DISMISSED** [review]
LGTM ✅

**Review by roystan-tm — APPROVED** [review]
lgtm

File: package.json (hunk 1 above)
Comment (NicholasValiente) [reply package.json:46]: ill have a look, but from testing it seems fine 👀

**Comment by NicholasValiente** [comment]
image
test with `service-sample-repo` worked https://github.com/cloudconformity/service-sample-repo/actions/runs/7548797230/job/20551575441

File: package.json (hunk 1 above)
Comment (NicholasValiente) [reply package.json:46]: @trend-adam-kinnell only that it dropped support for node 14 & 16
image

Comment (NicholasValiente) [reply package.json:46]: reverted this package to `^5.1.1` to not potentially break builds for now

**Review by trend-adam-kinnell — APPROVED** [review]
LGTM ✅

**Pull request merged**